snapcraft/tests/test_plugin_maven.py

```diff
@@ -57,6 +57,15 @@ def _canonicalize_settings(settings):
                 default_namespace='http://maven.apache.org/SETTINGS/1.0.0')
             return f.getvalue() + '\n'
 
+    def test_get_build_properties(self):
+        plugin = maven.MavenPlugin('test-part', self.options,
```

Comment:
a side comment, since this is a `classmethod` you can do
```python
expected_build_properties = ['maven-options', 'maven-targets']
resulting_build_properties = maven.MavenPlugin.get_build_properties()
self.assertThat(resulting_build_properties, HasLength(len(expected_build_properties))
for property in expected_build_properties:
    self.assertIn(property, resulting_build_properties)
```

Mind doing it like this? If yes you will need to add this import right bellow `import fixtures` so it looks like:

```python
import fixtures
from testtools.matchers import HasLength

import snapcraft
```